internal/auth0/organization/resource_connection.go

```diff
@@ -38,9 +37,24 @@ func NewConnectionResource() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 				Default:  false,
-				Description: "When true, all users that log in with this connection will be automatically granted " +
-					"membership in the organization. When false, users must be granted membership in the organization" +
-					" before logging in with this connection.",
+				Description: "When `true`, all users that log in with this connection will be automatically granted " +
+					"membership in the organization. When `false`, users must be granted membership in the organization " +
+					"before logging in with this connection.",
+			},
+			"is_signup_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
```suggestion
				Computed:  true,
```

We still need to set these as computed as opposed to having a default value.

Reply:
if we do, it breaks. The only way it works is if it is default. I am fairly sure it is a bug

Reply:
Indeed, it breaks the tests. For some reason, the default value allows the terraform state to work properly with the nil values. It also has the documentation advantage that it documents the actual behaviour more correctly